src/vs/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -166,6 +167,43 @@ export class NotebookEditor extends BaseEditor implements INotebookEditor {
 		this.editorExecutingNotebook = NOTEBOOK_EDITOR_EXECUTING_NOTEBOOK.bindTo(this.contextKeyService);
 	}
 
+	populateEditorTitlebar() {
+		for (let element: HTMLElement | null = this.rootElement.parentElement; element; element = element.parentElement) {
+			if (DOM.hasClass(element, 'editor-group-container')) {
+				// elemnet is editor group container
+				for (let i = 0; i < element.childElementCount; i++) {
+					const child = element.childNodes.item(i) as HTMLElement;
+
+					if (DOM.hasClass(child, 'title')) {
+						this.titleBar = child;
+						break;
+					}
+				}
+				break;
```

Comment:
@bpasero here I tried to get access to the title bar DOM node in order to change its z-index later on, is there any safer way to do this? Should we expose the DOM node through `IEditorGroupView.titlebarDomNode` etc?

Reply:
I am not a big fan of changing `z-index` to begin with. I am not a fan of doing this selectively based on the editor showing. Overall, -1. Please think about an alternative if possible.